examples/script_nifti.py

```diff
@@ -0,0 +1,10 @@
+from shimmingtoolbox.dicom_to_nifti import dicom_to_nifti
+
+def main():
+
+    unsortedDicomDir = r"C:\Users\gaspa\Desktop\Test\dicom_unsorted"
```

Comment:
Be carefull when using absolute paths. It is not reproductible for other users (You're the only pne with this forlder structure). Try to put a relative path so anyone can run your script/example or download live from a remote.

Reply:
Even more than that, any paths can be arguments passed on the command line, like this:

```
st_script_nifti C:\Users\gaspa\Desktop\Test\dicom_unsorted
```

For making reproducible tests, you can write a wrapper script that does something along these lines:

```
wget https://osf.io/something
unzip something.zip
st_script_nifti something\
```

Indeed, that's more or less the script we will be writing to do tests in CI.